src/commands/transaction/cancel.ts

```diff
@@ -0,0 +1,43 @@
+import { Argv } from "yargs";
+import { error, info } from "signale";
+import { getLogger } from "../../logger";
+import { cancelTransaction } from "../../implementations/transaction/transaction";
+import { TransactionCancelCommand } from "./transaction-command.type";
+import { withNetworkAndKeyOption } from "../shared";
+
+const { trace } = getLogger("transaction:cancel");
+
+export const command = "cancel [options]";
+
+export const describe = "Cancel pending transaction on the blockchain";
+
+export const builder = (yargs: Argv): Argv =>
+  withNetworkAndKeyOption(
+    yargs
+      .option("nonce", {
+        description: "Pending transaction nonce",
+        type: "string",
+        demandOption: true,
+      })
+      .option("gas", {
+        description: "Require higher gas fee than the pending transaction",
+        type: "string",
+        demandOption: true,
+      })
+  );
+
+export const handler = async (args: TransactionCancelCommand): Promise<void> => {
+  trace(`Args: ${JSON.stringify(args, null, 2)}`);
+  try {
+    if (!args.encryptedWalletPath) {
```

Comment:
we can support all the different way to provide a wallet, everything exist already to retrieve the wallet from the `withNetworkAndKeyOption`. This limit the usage to wallet path only

Reply:
Okay, i will remove the check since getWallet function will generate error if no key, keyFile or wallet path found